src/app/views/workflow-preview/WorkflowPreview.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import { push } from "react-router-redux";
+import { Link } from "react-router";
+import PropTypes from "prop-types";
+
+import url from "../../utilities/url";
+import notifications from "../../utilities/notifications";
+
+import Iframe from "../../components/iframe/Iframe";
+
+const propTypes = {
+    location: PropTypes.PropTypes.shape({
+        pathname: PropTypes.string
+    }),
+    params: PropTypes.shape({
+        collectionID: PropTypes.string.isRequired
+    }),
+    dispatch: PropTypes.func.isRequired
+};
+
+export class WorkflowPreview extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleBackButton = () => {
+        const previousUrl = url.resolve("../");
+        this.props.dispatch(push(previousUrl));
+    };
+
+    // getPreviewIframeURL returns the url for the content to be reviewed when the url is
+    // in the format: /florence/collections/collection-id/{content/to/preview/url}/preview
+    // returns '/' incase the url is '/homepage' or errors
+    getPreviewIframeURL = path => {
+        try {
+            const regex = /(?:\/[^/]+){3}(?<contentURL>.+)\/preview/;
+            const regexResult = regex.exec(path);
+            const contentURL = regexResult.groups.contentURL;
+            if (contentURL === "/homepage") {
+                return "/";
+            }
+            return contentURL;
+        } catch (error) {
+            notifications.add({
+                type: "warning",
+                message:
+                    "There was an error previewing this content so you've bene directed to the homepage. You can navigate to the content or refresh Florence.",
+                isDismissable: true
+            });
+            return "/";
+        }
+    };
+
+    render() {
+        this.getPreviewIframeURL(this.props.location.pathname);
+        return (
+            <div>
```

Comment:
Is it worth updating to use more semantic markup across the render method? It's something I need to be more mindful of when writing my components, too